.github/workflows/python-package.yml

```diff
@@ -0,0 +1,30 @@
+name: Python package
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [2.7, 3.5, 3.6]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 nose wheel
+          pip install waitress==1.4.0
+          pip install .[middleware]
+      - name: Test with nose
+        run: |
+          python -bb setup.py nosetests
```

Comment:
Here's an action you'd want to use to mirror the old travis behavior of coverage collection: https://github.com/codecov/codecov-action

Reply:
I wouldn't mind that, but that doesn't need to be in this PR.